From 3bfa01c92c495f277d131a4b43ceed146d3fb7f0 Mon Sep 17 00:00:00 2001
From: Weiheng Ni <weihengn@gmail.com>
Date: Tue, 14 May 2024 22:01:06 -0700
Subject: [PATCH] Enable robotpi os detection

---
 src/rospkg/os_detect.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/rospkg/os_detect.py b/src/rospkg/os_detect.py
index d1ead71..c7fb295 100644
--- a/src/rospkg/os_detect.py
+++ b/src/rospkg/os_detect.py
@@ -651,6 +651,7 @@ OS_OSX = 'osx'
 OS_POP = 'pop'
 OS_QNX = 'qnx'
 OS_RHEL = 'rhel'
+OS_ROBOTPI =  'robotpi'
 OS_ROCKY = 'rocky'
 OS_SLACKWARE = 'slackware'
 OS_UBUNTU = 'ubuntu'
@@ -692,6 +693,7 @@ OsDetect.register_default(OS_OSX, OSX())
 OsDetect.register_default(OS_POP, LsbDetect("Pop"))
 OsDetect.register_default(OS_QNX, QNX())
 OsDetect.register_default(OS_RHEL, FdoDetect("rhel"))
+OsDetect.register_default(OS_ROBOTPI, FdoDetect("robotpi"))
 OsDetect.register_default(OS_ROCKY, FdoDetect("rocky"))
 OsDetect.register_default(OS_SLACKWARE, Slackware())
 OsDetect.register_default(OS_UBUNTU, LsbDetect("Ubuntu"))
